rust/build.rs

```diff
@@ -0,0 +1,56 @@
+#[cfg(all(target_os = "linux", target_env = "gnu"))]
+mod platform_cfg {
+    // glibc version is taken from std/sys/unix/os.rs
+    pub fn glibc_version() -> (usize, usize) {
+        use regex::Regex;
+        use std::process::Command;
+
+        let output = Command::new("ldd")
+            .args(&["--version"])
+            .output()
+            .expect("failed to execute ldd");
+        let output_str = std::str::from_utf8(&output.stdout).unwrap();
+
+        let version_reg = Regex::new(r#"ldd \(.+\) ([0-9]+\.[0-9]+)"#).unwrap();
```

Comment:
Maybe you can add a unit-test for extracting the regex with an example of the glibc version?

Reply:
build.rs doesn't support unit test, let me pull those code into its own crate.

Reply:
@fvaleye moved that to a different crate and added tests.